server/db/Seeder.js

```diff
@@ -1,15 +1,32 @@
 import pg from "pg"
 import path from "path"
-import LineReader from "line-reader"
+//import LineReader from "line-reader"
 import { fileURLToPath } from "url"
+import fs from 'fs'
 
 const pool = new pg.Pool({
-  connectionString: "postgres://postgres:password@localhost:5432/<yourDBnameHere>" })
+  connectionString: "postgres://postgres:password@localhost:5432/pets-app" });
 
 class Seeder {
   static async seed() {
-    // your seeder code here
+    let loadedData = fs.readFileSync('data.json');
+    let contents = JSON.parse(loadedData);
```

Comment:
Serge - Here is that simple JSON data from file import I was thinking of: const data = require('.filePath/filename.json');
console.log(data); Nothing to change here, I was just really struck by this style of JSON data import when it came up in one of our mentor sessions. It lets us slide on import fs from 'fs'.